hackvote/app/models/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event < ActiveRecord::Base
+  has_many :projects
+  # order so we can display past hackdays
+  default_scope -> { order(created_at: :desc) }
+  # make sure we have title for event, and title doesn't overflow string.
+  validates :title, presence: true, length: { maximum: 255 }
+
+  def project_feed
+  	Project.where("event_id = ?", id)
```

Comment:
I was wondering why we had to add the has_many property to models. Does this mean rails has a built in function to retrieve projects that belong to a certain event?